### PR TITLE
#155587846 chore: update model ids to cuids

### PR DESCRIPTION
**Pull request by Mercy-Muchai**
#### What does this PR do?
Updates model ids to cuids
#### Description of Task to be completed?
- update model ids
- update migration ids
- update locations seeders
- test endpoints
- rm Alias for Assignees
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[155587846](https://www.pivotaltracker.com/story/show/155587846)
#### Screenshots (if appropriate)
screen shot 2018-03-05 at 1 16 57 pm

### CODE REVIEW

**Comment by Mercy-Muchai** [comment]
@kimobrian Updated this PR to only take changes on IDs not inclusive of the other open PR

File: package.json [hunk 1]
```diff
@@ -40,5 +27,13 @@
     "pg-hstore": "^2.3.2",
     "sequelize": "^3.30.4",
     "sequelize-cli": "^2.7.0"
```
Comment (Mnickii) [inline package.json]: add cuid to the dependencies

Comment (Mercy-Muchai) [reply package.json]: Updated

**Comment by Mnickii** [comment]
LGTM

**Pull request merged**